README.md

```diff
@@ -33,26 +33,29 @@ Use `npm run test:feature` to run E2E tests with the Cypress GUI or simply use `
 
 ### Running the linter
 
-Solidus NextJS Frontent uses prettier-standard library as style guide, linter and formatter.
-It works as a standlone system and doens't need an `.eslintrc` file.
+Solidus NextJS Frontent uses eslint with the standardjs rule and prettier libraries
+as style guide, linter and formatter.
```

Comment:
`Frontent` -> `Frontend`

`eslint with the standardjs rule` -> `ESLint with the JavaScript Standard style guide and Prettier as formatter`.

Is the above more correct?